swap_meet/vendor.py

```diff
@@ -1,2 +1,101 @@
+from swap_meet.item import Item
+
+# Wave 1
+"""The class vendor takes in a parameter of inventory. It has a method function
+    that adds and removes items.
+    Returns:
+      It returns the item or the boolean false.
+        """
+
+
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+        self.item = Item()
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
`else` clauses tend to be used as a last-case scenario in case no other conditions were met. Right now, the primary logic of removing an item is indented and not a center-focus of this function. To make this logic more apparent, we can use the pattern of a guard clause by reversing the condition: 

```suggestion
    def remove(self, item):
        if item not in self.inventory:
            return False
            
        self.inventory.remove(item)
        return item
```